client/java/kserve-api/resnet_labels.txt

```diff
@@ -0,0 +1,1000 @@
+tench, Tinca tinca
```

Comment:
We already duplicate those labels in multiple places:
https://github.com/openvinotoolkit/model_server/search?q=magpie

Can we do something about it to avoid that?